src/com/owncloud/android/ui/helpers/FileOperationsHelper.java

```diff
@@ -134,6 +136,37 @@ public void openFile(OCFile file) {
     }
 
     /**
+     * Show dialog to allow the user to choose an app to send the private link of an {@link OCFile},
+     * or copy it to clipboard.
+     *
+     * @param context
+     * @param file @param file {@link OCFile} which will be shared with internal users
+     * @param account
+     */
+    public void copyOrSendPrivateLink(Context context, Account account, OCFile file) {
```

Comment:
There is no need to pass a context to the method. ```FileOperationsHelper``` has a field ```mFileActivity``` that can be used instead.